modules/cluster-logging-systemd-scaling.adoc

```diff
@@ -17,7 +17,7 @@ and other settings.
 
 .Procedure
 
-. Create a `journald.conf` file with the required settings:
+. Create a `journald.conf` file with the required settings by running the following command:
```

Comment:
In case of file contents, shall I also go ahead and remove the attributes for styling below? Currently it says [source, terminal]. Can be kept as a plain code block

Reply:
The docs guidelines for [code blocks](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output) is not explicitly clear regarding configuration files.  However, because command output has explicit guidance to use `[source,terminal]`, I'm prepared to infer that it's plausible that customers can create text files using a terminal editor like vi, emacs, other personal preference.  My sugg is to keep the `[source,terminal]`.